src/main/java/com/mojang/brigadier/arguments/LongArgumentType.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package com.mojang.brigadier.arguments;
+
+import com.mojang.brigadier.StringReader;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public class LongArgumentType implements ArgumentType<Long> {
+    private static final Collection<String> EXAMPLES = Arrays.asList("0", "123", "-123");
```

Comment:
You don't declare `getExamples`, so this ends up being unused.

Reply:
Same in `IntegerArgumentType` also, I just copied from that and didn't notice. Both need to be fixed.

Reply:
`IntegerArgumentType` was already fixed, though after you commit was merged: https://github.com/Mojang/brigadier/blob/master/src/main/java/com/mojang/brigadier/arguments/IntegerArgumentType.java#L90

Reply:
See #16